src/connectedk8s/setup.py

```diff
@@ -17,7 +17,7 @@
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
 
-VERSION = '1.5.5'
+VERSION = '1.5.4'
```

Comment:
Please do not downgrade the extension version. When you need to release a new extension version the next time, just upgrade to the new version.

Reply:
Updated